src/6_branch.js

```diff
@@ -398,6 +398,12 @@ Branch.prototype['init'] = wrap(
 			}
 
 			var additionalMetadata = utils.getAdditionalMetadata();
```

Comment:
You should update getAdditionalMetadata to return an empty object instead of null if there is no hosted deeplink data so you don't have to do the extra check on line 403